cartography/intel/aws/resourcegroupstaggingapi.py

```diff
@@ -48,6 +66,8 @@ def get_bucket_name_from_arn(bucket_arn: str) -> str:
     'ec2:vpc': {'label': 'AWSVpc', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
     'ec2:transit-gateway': {'label': 'AWSTransitGateway', 'property': 'id'},
     'ec2:transit-gateway-attachment': {'label': 'AWSTransitGatewayAttachment', 'property': 'id'},
+    'elasticloadbalancing:loadbalancer': {'label': 'LoadBalancer', 'property': 'name', 'id_func': get_short_id_from_elb_arn},
+    'elasticloadbalancing:loadbalancer/app': {'label': 'LoadBalancerV2', 'property': 'name', 'id_func': get_short_id_from_alb_arn},
```

Comment:
[question] Hey, so I didn't write our ELBv2 code and I don't know load balancers in AWS really well, but I see that ELBv2s can either be have `loadbalancer/app` or `loadbalancer/net` in the resource name: https://docs.aws.amazon.com/service-authorization/latest/reference/list_elasticloadbalancingv2.html.

Our [schema doc](https://github.com/lyft/cartography/blob/master/docs/schema/aws.md#loadbalancerv2) says we support both ALBs and NLBs, and a quick look at the [code](https://github.com/lyft/cartography/blob/master/cartography/intel/aws/ec2/load_balancer_v2s.py) seems to confirm this.

Should also add another line for

```python
'elasticloadbalancing:loadbalancer/net': {'label': 'LoadBalancerV2', 'property': 'name', 'id_func': get_short_id_from_nlb_arn},
```
?